kernel/src/main/scala/org/apache/toree/kernel/api/Kernel.scala

```diff
@@ -20,8 +20,11 @@ package org.apache.toree.kernel.api
 import java.io.{InputStream, PrintStream}
 import java.net.URI
 import java.util.concurrent.{ConcurrentHashMap, TimeUnit, TimeoutException}
+
```

Comment:
Nit: unnecessary changes in imports make backports and branch maintenance harder. I'd prefer not to have these changes unless there is a stated style that this changes conforms to.